ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -200,11 +483,44 @@ export default class SecurityTab extends PureComponent {
     return (
       <div className="settings-page__body">
         {warning ? <div className="settings-tab__error">{warning}</div> : null}
-        {this.renderSeedWords()}
-        {this.renderIncomingTransactionsOptIn()}
-        {this.renderPhishingDetectionToggle()}
+        <span className="settings-page__security-tab-sub-header__bold">
+          {this.context.t('security')}
+        </span>
+        <div className="settings-page__content-padded">
+          {this.renderSeedWords()}
+        </div>
+        <span className="settings-page__security-tab-sub-header__bold">
+          {this.context.t('privacy')}
+        </span>
+        <div>
+          <span className="settings-page__security-tab-sub-header">Alerts</span>
+        </div>
+        <div className="settings-page__content-padded">
+          {this.renderPhishingDetectionToggle()}
+        </div>
+
+        <span className="settings-page__security-tab-sub-header">
+          {this.context.t('transactions')}
+        </span>
+        <div className="settings-page__content-padded">
+          {this.renderIncomingTransactionsOptIn()}
+        </div>
+        <span className="settings-page__security-tab-sub-header">
+          {this.context.t('networkProvider')}
+        </span>
+        <div className="settings-page__content-padded">
+          {this.renderChooseYourNetworkButton()}
+          {this.renderIpfsGatewayControl()}
+        </div>
+        <span className="settings-page__security-tab-sub-header">
+          {this.context.t('tokenNftAutoDetection')}
+        </span>
+        <div className="settings-page__content-padded">
+          {this.renderAutoDectectTokensToggle()}
+          {this.renderBatchAccountBalanceRequestsToggle()}
+          {this.renderCollectibleDetectionToggle()}
+        </div>
         {this.renderMetaMetricsOptIn()}
```

Comment:
I believe we're missing the title `Usage` above this line.

Reply:
@darkwing if you mean the `<span className="settings-page__security-tab-sub-header">`, it was decided that we should leave this like that.